core/src/test/java/org/opensearch/sql/analysis/AnalyzerTest.java

```diff
@@ -274,7 +292,7 @@ public void project_values() {
         AstDSL.project(
             AstDSL.values(ImmutableList.of(AstDSL.intLiteral(123))),
             AstDSL.alias("123", AstDSL.intLiteral(123)),
-            AstDSL.alias("hello", AstDSL.stringLiteral("hello")),
+            AstDSL.alias("hello", stringLiteral("hello")),
```

Comment:
Why?

Reply:
I believe this was an accidental change done in a previous draft PR. Should be fixed under [57ec0691b36f1e0010c034df1a59ef6b364feaf4](https://github.com/Bit-Quill/opensearch-project-sql/pull/96/commits/57ec0691b36f1e0010c034df1a59ef6b364feaf4).